plugins/pulp_rpm/plugins/importers/iso/sync.py

```diff
@@ -229,7 +234,9 @@ def perform_sync(self):
                     iso.save()
                 except NotUniqueError:
                     iso = iso.__class__.objects.filter(**iso.unit_key).first()
-                self.add_catalog_entries([iso])
+                else:
+                    # Only if iso is from pulp manifest
```

Comment:
Right.  I understand what the code is doing (and why).  It's very straight forward.  It's the comment that I find confusing.  Suggest removing it.